### ISSUE TITLE
Terrible sound regression on LBA2 (like on latest dosbox) on both windows and linux

### ISSUE THREAD

**Issue by Askmewho** [joncampbell123/dosbox-x#1404]
**Describe the bug**
the music sound horrible, especially on the menu

**To Reproduce**
1)go to the menu

**Expected behavior**
not to happen hhaha

**Environment (please complete the following information):**
both linux and windows

**Comment by joncampbell123** [comment]
Please spell out the full title of LBA2 and the company and publisher behind it.

**Comment by Askmewho** [comment]
oh ,sorry. It's little big adventure 2

**Comment by Allofich** [comment]
Seems like this game uses CD audio for the music, so I'm guessing "horrible" means it's skipping or something? @Askmewho I think you need to give more information, like what does "horrible" mean (can you make a sound recording?) and what format is the game data in? Are you playing the game using a CD (could be a problem with the CD?) or is it from Steam or GOG.com or something?

**Comment by Askmewho** [comment]
I mean regressions bc its present on latest dosbox but isn't on older dosbox, don't know about older dosbox-x. Ive use an iso. I will record it but its just heavily distorted sound.

**Comment by Allofich** [comment]
The problem is in latest DOSBox (DOSBox 0.74-3?), but not in older DOSBox versions? Does the same problem happen in latest DOSBox-X and DOSBox 0.74-3? It helps to know the versions and whether the problem is something in regular DOSBox or if it is something that only happens in DOSBox-X.

>I will record it but its just heavily distorted sound.
It still may be useful.

**Comment by Allofich** [comment]
Actually, the demo does have in-game music, and it plays fine.

> but menu and some music on the games are heavily distorted

What is the "menu"? When you press ESC during in-game, or when you start the game and it offers to let you continue your old game or start a new one? These are both silent in the demo.

When I searched for footage of this game, I found a video with the start menu and it was silent there, too. The same video showed regular in-game music playing normally. I haven't found footage showing when the menu when the ESC key is pressed.
